app/helpers/picture_version_helper.rb

```diff
@@ -35,7 +35,7 @@ def picture_version(picture, settings = {})
   # @param version_picture [Alchemy::PictureVersion]
   def picture_version_url(version)
     if Dragonfly.app(:alchemy_pictures).datastore.is_a?(Dragonfly::S3DataStore)
-      Dragonfly.app(:alchemy_pictures).datastore.url_for(version.file_uuid)
+      Dragonfly.app(:alchemy_pictures).datastore.url_for(version.file_uuid).sub(%r(^http:\/\/), 'https://')
```

Comment:
`/` characters do not need to be escaped in a regex literal, and `^` matches line start whereas `\A` matches string start, i.e. change this to:
```ruby
.sub(%r(\Ahttp://), 'https://')
```